modules/auxiliary/scanner/ssl/ssl_version.rb

```diff
@@ -472,7 +472,7 @@ def run_host(ip)
     begin
       ctx = { 'Msf' => framework, 'MsfExploit' => self }
       # Initialize rex-sslscan scanner
-      scanner = Rex::SSLScan::Scanner.new(ip, rport, ctx)
+      scanner = Rex::SSLScan::Scanner.new(ip, rport, ctx, tls_server_name_indication: datastore['RHOSTNAME'])
```

Comment:
Apologies, my brain autocompleted `datastore['RHOSTNAME']` as `datastore['SSLServerNameIndication']`.

For context, we've got an existing convention of `SSLServerNameIndication` that you'll want to follow - that's what my original comment was aimed at following :+1:

https://github.com/search?q=repo%3Arapid7%2Fmetasploit-framework%20SSLServerNameIndication&type=code